hazelcast/src/test/java/com/hazelcast/client/impl/MemberImplTest.java

```diff
@@ -0,0 +1,196 @@
+package com.hazelcast.client.impl;
+
+import com.hazelcast.nio.Address;
+import com.hazelcast.test.HazelcastParallelClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.ParallelTest;
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(HazelcastParallelClassRunner.class)
+@Category({QuickTest.class, ParallelTest.class})
+public class MemberImplTest extends HazelcastTestSupport {
+
+    private static Address address;
+
+    private MemberImpl member;
```

Comment:
I'm always very careful putting in such a field in my test. For example the method 'assertMemberImpl' makes no sense until you realize that it is about the field member.